forms/jqm/element.js

```diff
@@ -158,6 +166,10 @@ define(function (require) {
 
     },
 
+    onChangeClass: function () {
+      this.$el.attr('class', this.model.attributes['class']);
```

Comment:
Actually, I think I have a way to back-up any new DOM classes during this handler without going crazy.